call-home/src/main.rs

```diff
@@ -50,7 +50,7 @@ async fn main() {
 
 async fn run() -> anyhow::Result<()> {
     let args = CliArgs::args();
-    let version = clap::crate_version!().to_string();
+    let version = "2.0.1".to_string();
```

Comment:
We may not need to change this. Below change for CAStor version should be sufficient.

Reply:
We are using this... CAstor version is left in because to header requirements by the server.

Reply:
I could bump the crate version, if that is preferred.